lib/src/signed_video_internal.h

```diff
@@ -156,6 +156,8 @@ struct _signed_video_t {
   // Frame counter and flag to handle recurrence
   bool has_recurrent_data;
   int frame_count;
+  uint8_t received_gop_hash[MAX_HASH_SIZE];  // Received hash list after decoding SEI data while
```

Comment:
Put this among the other validation specific members further down. For example, after `authentication_started`